src/module.ts

```diff
@@ -32,167 +31,129 @@ const nuxtModule: Module<ModuleOptions> = function (moduleOptions) {
-        hid: 'gf-style',
-        rel: 'stylesheet',
-        href: url
-      })
+      // https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch#Best_practices
+      // connect to domain of font files
+      if (moduleOptions.preconnect) {
+        nuxt.options.head.link.push({
+          hid: 'gf-preconnect',
+          rel: 'preconnect',
+          href: 'https://fonts.gstatic.com/',
+          crossorigin: ''
+        })
 
-      return
-    }
+        // Should also preconnect to origin of Google fonts stylesheet.
+        nuxt.options.head.link.push({
+          hid: 'gf-origin-preconnect',
+          rel: 'preconnect',
+          href: 'https://fonts.googleapis.com/'
+        })
+      }
 
-    // JS to inject CSS
-    // @ts-ignore
-    this.options.head.script = this.options.head.script || []
-    // @ts-ignore
-    this.options.head.script.push({
-      hid: 'gf-script',
-      innerHTML: `(function(){var l=document.createElement('link');l.rel="stylesheet";l.href="${url}";document.querySelector("head").appendChild(l);})();`
-    })
+      // https://developer.mozilla.org/pt-BR/docs/Web/HTML/Preloading_content
+      // optionally increase loading priority
+      if (moduleOptions.preload) {
+        nuxt.options.head.link.push({
+          hid: 'gf-preload',
+          rel: 'preload',
+          as: 'style',
+          href: url
+        })
+      }
 
-    // no-JS fallback
-    // @ts-ignore
-    this.options.head.noscript = this.options.head.noscript || []
-    // @ts-ignore
-    this.options.head.noscript.push({
-      hid: 'gf-noscript',
-      innerHTML: `<link rel="stylesheet" href="${url}">`
-    })
+      // append CSS
+      if (moduleOptions.useStylesheet) {
+        // @ts-ignore
+        nuxt.options.head.link.push({
+          hid: 'gf-style',
+          rel: 'stylesheet',
+          href: url
+        })
 
-    // Disable sanitazions
-    // @ts-ignore
-    this.options.head.__dangerouslyDisableSanitizersByTagID = this.options.head.__dangerouslyDisableSanitizersByTagID || {}
-    // @ts-ignore
-    this.options.head.__dangerouslyDisableSanitizersByTagID['gf-script'] = ['innerHTML']
-    // @ts-ignore
-    this.options.head.__dangerouslyDisableSanitizersByTagID['gf-noscript'] = ['innerHTML']
-  })
-}
+        return
+      }
 
-;(nuxtModule as any).meta = { name, version }
+      // JS to inject CSS
+      nuxt.options.head.script!.push({
+        hid: 'gf-script',
+        innerHTML: `(function(){var l=document.createElement('link');l.rel="stylesheet";l.href="${url}";document.querySelector("head").appendChild(l);})();`
+      })
 
-declare module '@nuxt/types' {
-  interface NuxtConfig { [CONFIG_KEY]?: ModuleOptions } // Nuxt 2.14+
-  interface Configuration { [CONFIG_KEY]?: ModuleOptions } // Nuxt 2.9 - 2.13
-}
+      // no-JS fallback
+      nuxt.options.head.noscript!.push({
+        hid: 'gf-noscript',
+        innerHTML: `<link rel="stylesheet" href="${url}">`
+      })
```

Comment:
Noscript is not implemented yet for vueuse/head - but see https://github.com/vueuse/head/pull/71.

Reply:
Ok, added comment